plugin/src/lib.rs

```diff
@@ -724,6 +724,18 @@ fn fold_first_block(block: P<Block>, m: &mut MutatorPlugin) -> P<Block> {
                         .unwrap(),
                 );
             }
+            let (ref key, ref values) = for method_info in info.method_infos {
+                for interchangeable in method_info.interchangeables {
+                    let ident = interchangeable.name.to_ident();
```

Comment:
@llogiq - I still need your help. When I try to compile the code I'm getting error.

```

//let ($a, $b) = if mutagen::now($n) { ($b, $a) } else { ($a, $b) };
             //TODO: switch interchangeables, need mutability info, too
             //for name in method_info.interchangeables { }

```

Do I've to move the for loop inside the if [](https://github.com/llogiq/mutagen/blob/master/plugin/src/lib.rs#L695)